### PR TITLE
Update template for README.md file

### PR DESCRIPTION
**Pull request by wiegandm**
## What

The template for the `README.md` was updated with current requirements and made more concise.

### CODE REVIEW

File: README-template.md [hunk 1]
```diff
@@ -1,94 +1,42 @@
...
-Code blocks illustrating common usage with explanation.
-
-For CLI tools, prefix commands with `$ ` and include example output.
-
-Use a paragraph instead of comment inside code block for explanation.
-
-For longer examples, also include runnable script and link to it here.
-
-Link to longer documentation if available here.
+Optionally include one screenshot if the component has a user interface.
```
Comment (k-schlosser) [inline README-template.md:18]:
```suggestion
Optionally, include one screenshot if the component has a user interface.
```

File: README-template.md [hunk 2]
```diff
@@ -1,94 +1,42 @@
...
 
 ## Support
 
-For any question on the usage of PROJECT please use the [Greenbone Community Portal](https://community.greenbone.net/c/X). If you found a problem with the software, please [create an issue](https://github.com/greenbone/PROJECT/issues) on GitHub. If you are a Greenbone customer, you may alternatively or additionally forward your issue to the Greenbone Support Portal.
-
-## Maintainer
+Include information on how users can receive support, for example:
 
-This project is maintained by [Greenbone Networks GmbH](https://www.greenbone.net/).
+For any question on the usage of COMPONENT please use the [Greenbone Community Portal](https://community.greenbone.net/c/X). If you found a problem with the software, please [create an issue](https://github.com/greenbone/COMPONENT/issues) on GitHub. If you are a Greenbone customer, you may alternatively or additionally forward your issue to the Greenbone Support Portal.
```
Comment (k-schlosser) [inline README-template.md:24]:
```suggestion
For any question on the usage of COMPONENT, please use the [Greenbone Community Portal](https://community.greenbone.net/c/X). If you found a problem with the software, please [create an issue](https://github.com/greenbone/COMPONENT/issues) on GitHub. If you are a Greenbone customer, you may alternatively or additionally forward your issue to the [Greenbone Support Portal](https://jira.greenbone.net/servicedesk/customer/portal/7).
```

File: README-template.md [hunk 3]
```diff
@@ -1,94 +1,42 @@
...
-
-    pip install --user pipenv
-
-Afterwards run
-
-    pipenv install --dev
+Include information on how users can contribute, for example:
 
-in the checkout directory of PROJECT (the directory containing the Pipfile) to install all dependencies including the packages only required for development.
+This project is maintained by [Greenbone AG](https://www.greenbone.net/).
```
Comment (k-schlosser) [inline README-template.md:30]:
```suggestion
This project is maintained by the [Greenbone AG](https://www.greenbone.net/).
```

File: README-template.md [hunk 4]
```diff
@@ -1,94 +1,42 @@
...
 
 ## License
 
-Copyright (C) 2009-2018 [Greenbone Networks GmbH](https://www.greenbone.net/)
+Copyright (C) 2023 [Greenbone AG](https://www.greenbone.net/)
 
 Licensed under the [GNU General Public License v3.0 or later](LICENSE).
 
-State *license name* and *license owner*. Use the [SPDX](https://spdx.org/licenses/) *full name* or *identifier* for the license name and link to included `LICENSE` file.
+State *license name* and *copyright owner*. Use the [SPDX](https://spdx.org/licenses/) *full name* or *identifier* for the license name and link to included `LICENSE` file containing the license text.
```
Comment (k-schlosser) [inline README-template.md:42]:
```suggestion
State *license name* and *copyright owner*. Use the [SPDX](https://spdx.org/licenses/) *full name* or *identifier* for the license name and link to the included `LICENSE` file containing the license text.
```

File: README-template.md [hunk 5]
```diff
@@ -1,94 +1,42 @@
...
 
-[![GitHub releases](https://img.shields.io/github/release/greenbone/PROJECT.svg)](https://github.com/greenbone/PROJECT/releases)
-[![PyPI release](https://img.shields.io/pypi/v/PROJECT.svg)](https://pypi.org/project/PROJECT/)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/greenbone/PROJECT/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/greenbone/PROJECT/?branch=master)
-[![code test coverage](https://codecov.io/gh/greenbone/PROJECT/branch/master/graph/badge.svg)](https://codecov.io/gh/greenbone/PROJECT)
-[![CircleCI](https://circleci.com/gh/greenbone/PROJECT/tree/master.svg?style=svg)](https://circleci.com/gh/greenbone/PROJECT/tree/master)
+![Greenbone Logo](https://www.greenbone.net/wp-content/uploads/Gb_New-logo_horizontal_head.png)
 
-TODO: Replace PROJECT with project references!
+# Greenbone Software Component
```
Comment (k-schlosser) [inline README-template.md:9]: I think this should be a placeholder? Can we make that more obvious?

**Review by k-schlosser — CHANGES_REQUESTED** [review]

**Review by y0urself — APPROVED** [review]

File: README-template.md (hunk 5 above)
Comment (y0urself) [reply README-template.md:9]:
```suggestion
# Greenbone <SOFTWARE COMPONENT>
```

Maybe like this?

Comment (k-schlosser) [reply README-template.md:9]: Yes, looks good.

**Review by k-schlosser — APPROVED** [review]

**Pull request merged**